src/main/java/core/basesyntax/db/FileReaderImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.db;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    public FileReaderImpl(String s) {
+    }
+
+    public List<String> read(String fileName) {

```

Comment:
The method parameter 'fileName' is not validated. Consider adding null check or empty string check.

Reply:
ok, done